builder/lxc/step_lxc_create.go

```diff
@@ -88,6 +104,16 @@ func (s *stepLxcCreate) Run(state multistep.StateBag) multistep.StepAction {
 		errorHandler(err)
 		return multistep.ActionHalt
 	}
+
+  for _, sidedisk := range config.SidediskFolders {
+    ui.Say(fmt.Sprintf("Loading sidedisk \"%s\" to %s", sidedisk.Archive, sidedisk.Dest))
+    err = s.loadSidedisk(config.ContainerName, sidedisk.Archive, sidedisk.Dest)
+    if err != nil {
+  		errorHandler(err)
+  		return multistep.ActionHalt
+  	}
+  }
+
```

Comment:
Do we want to wait for the script to try to unpack all the sidedisk or de we want to halt at the first error ?